tests/integration/targets/inventory_kubevirt/playbook.yml

```diff
@@ -0,0 +1,62 @@
+- name: Create VM
+  connection: local
+  gather_facts: false
+  hosts: localhost
+  tasks:
+    - name: Create a VirtualMachine
+      kubernetes.kubevirt.kubevirt_vm:
+        state: present
+        name: testvm
+        namespace: default
+        labels:
+          app: test
+        wait: yes
+        wait_timeout: 600
+        spec:
+          domain:
+            cpu:
+              sockets: 1
+            memory:
+              guest: 256Mi
```

Comment:
This is very low memory for Fedora, can we increase it a bit to ensure nothing crashes because of OOM?